src/components/accordion/_accordion.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+
+@include govuk-exports("govuk/component/accordion") {
+
+  .govuk-accordion__controls {
+    text-align: right;
+  }
+
+  // This is styled to look like a link not a button
+  .govuk-accordion__expand-all {
+    @include govuk-font($size: 16);
+    display: inline;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: transparent;
+    cursor: pointer;
+  }
+
+  // Hide the body of collapsed sections
+  .govuk-accordion__section[aria-expanded="false"] .govuk-accordion__section-body {
+    display: none;
+  }
+
+  // Border at the bottom of the whole accordtion
+  .govuk-accordion.with-js {
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+    border-bottom-color: $govuk-border-colour;
+  }
+
+  // Borders between accordion sections
+  .govuk-accordion.with-js .govuk-accordion__section {
```

Comment:
👍 on switching to `js-enabled` from `with-js` for consistency with existing components.

I'm happy to make use of SASS nesting too – wasn't sure from your [coding standards](https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/coding-standards/css.md#nesting) whether that was encouraged or not.

Reply:
Yeah we decided against SASS nesting for readability, it's documented [here](https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/coding-standards/css.md#nesting). It feels like it's one of those situations where there are benefits to both approaches. At the same time I think in the long run this will make the codebase easier to maintain.

Reply:
@frankieroberto Did you get a chance to look into switching this to use `js-enabled`? It would be good if it was consistent with the other components.